source/adapters/level_zero/command_buffer.cpp

```diff
@@ -699,11 +699,12 @@ urCommandBufferRetainExp(ur_exp_command_buffer_handle_t CommandBuffer) {
 }
 
 UR_APIEXPORT ur_result_t UR_APICALL
-urCommandBufferReleaseExp(ur_exp_command_buffer_handle_t CommandBuffer) {
+urCommandBufferReleaseExp(ur_exp_command_buffer_handle_t &CommandBuffer) {
   if (!CommandBuffer->RefCount.decrementAndTest())
     return UR_RESULT_SUCCESS;
 
   delete CommandBuffer;
+  CommandBuffer = nullptr;
```

Comment:
It is not the responsibility of the level zero adapter to zero out the handle pointer as part of the `urCommandBufferReleaseExp` entry point. The proper fix for this should be in the layer above which is calling this entry point.